collectors/python.d.plugin/sslcheck/README.md

```diff
@@ -0,0 +1,9 @@
+### configuration
+
+```yaml
+master:
+  host: www.my-netdata.io
+  port: 443
+  timeout: 10
+```
+
```

Comment:
please add more info to the README

Description, requirements if any, what charts it produces, defaults values etc, etc.

[example](https://github.com/netdata/netdata/tree/master/collectors/python.d.plugin/mongodb)